WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -496,6 +501,31 @@ extension NotificationDetailsViewController {
 
 
 
+// MARK: - Reader Helpers
+//
+private extension NotificationDetailsViewController {
+    func attachReaderViewIfNeeded() {
+        guard shouldAttachReaderView,
+            let postID = note.metaPostID,
+            let siteID = note.metaSiteID else {
+                readerDetailViewController?.remove()
+            return
```

Comment:
`return` needs indentation 😬

Reply:
Ops! Fixed in fc8ff0d8533620c0e6f6d2b678971761df5f29a3